ci/run_phpstan.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# -x Exit immediately if any command fails
+# -e Echo all commands being executed.
+# -u Fail for undefined variables
+set -xeu
+# This installs and runs phpstan
+# TODO: Remove separate phpstan install step after https://github.com/microsoft/tolerant-php-parser/pull/385 is merged
```

Comment:
I've merged 385 and removed a merge conflict with main, do you want to remove this now?

Reply:
This separate step can be removed now though right?